content/2021/2021-04/2021-04-26-pulsar-weekly.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "2021-04-26-pulsar-weekly"
+displayTitle: "Pulsar Community Weekly Update - 2021-04-19 ~ 2021-04-25"
+date: 2021-04-19 ~ 2021-04-25
+description: "This is the Pulsar community weekly update for 2021-04-19 ~ 2021-04-25, with updates on Pulsar client, broker, transactions, and so on."
+id: "2021-04-26-pulsar-weekly"
+---
+
+## [Pulsar Community Weekly Update] 2021-04-19 ~ 2021-04-25
+
+This is the Pulsar community weekly update for 2021-04-19 ~ 2021-04-25, with updates on Pulsar client, broker, transactions, and so on.
+
+### Pulsar Highlight
+
+- Add some basic supports for testing de-duplication feature.
```

Comment:
Do we add PR link for it?

Reply:
based on our template, Pulsar Highlight part does not cover the PR link.

Reply:
If we have PR or PIP to track it, it's better to add it.
You can discuss with eng and decide.